internal/manager/run.go

```diff
@@ -132,18 +132,19 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 	setupLog.Info("Initializing Dataplane Client")
 	eventRecorder := mgr.GetEventRecorderFor(KongClientEventRecorderComponentName)
 
+	readinessChecker := clients.NewDefaultReadinessChecker(adminAPIClientsFactory, deprecatedLogger)
```

Comment:
Like in the previous comment, we do not want to increase the usage of `deprecatedLogger`.